CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Change Log
 
+### v1.1.0
```

Comment:
There's a large amount of other changes that are missing from the 1.1.0 changelog. Should we maybe move this part to a separate PR?

Reply:
Actually this change didn't make 1.1, so I will rev it to 1.2.0 and add a bit about 1.1.0. Most of 1.1.0 was bug fixes.